### PR TITLE
Replace references of openjdk-systemtest with aqa-systemtest

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Related to https://github.com/adoptium/aqa-systemtest/issues/424

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Works fine in non-parallel build. Sanity.jck green: Test_grinder/job/Grinder/14986/

File: system/README.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
 System tests help verify that the AdoptOpenJDK binaries are *good* by running a variety of load tests. 
 
 ## Related repositories: 
-- System tests that run both on AdoptOpenJDK and AdoptOpenJDK-OpenJ9 SDKs are located at:  github.com/AdoptOpenJDK/openjdk-systemtest.git
+- System tests that run both on AdoptOpenJDK and AdoptOpenJDK-OpenJ9 SDKs are located at:  github.com/AdoptOpenJDK/aqa-systemtest.git
```
Comment (llxia) [inline system/README.md:6]: AdoptOpenJDK -> adoptium

File: system/README.md [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
 - System tests specific to AdoptOpenJDK-OpenJ9 are located at: github.com/eclipse-openj9/openj9-systemtest.git
 - STF framework that runs system tests are located at: github.com/AdoptOpenJDK/stf.git
 
 ## How it works
 The [build.xml](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/system/build.xml) file defines steps to pull test materials from the above 3 repositories, and compile them. The [playlist.xml](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/system/playlist.xml) file defines a set of test we want to run. 
 
 For detailed documentation, please see 
-- openjdk-systemtest [build instruction](https://github.com/AdoptOpenJDK/openjdk-systemtest/blob/master/openjdk.build/docs/build.md) 
+- aqa-systemtest [build instruction](https://github.com/AdoptOpenJDK/aqa-systemtest/blob/master/openjdk.build/docs/build.md)
```
[the remark this answers is not in the document]
Comment (Mesbah-Alam) [reply system/README.md]: Fixed.

File: buildenv/jenkins/JenkinsfileBase [hunk 3]
```diff
@@ -398,7 +398,7 @@ def setup() {
 
 			// system test repository exports to be used by system/common.xml
 			if (params.ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH) {
-				String[] adoptSystemTest = getGitRepoBranch(params.ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH, "AdoptOpenJDK:master", "openjdk-systemtest")
+				String[] adoptSystemTest = getGitRepoBranch(params.ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH, "AdoptOpenJDK:master", "aqa-systemtest")
```
[the remark this answers is not in the document]
Comment (Mesbah-Alam) [reply buildenv/jenkins/JenkinsfileBase]: Fixed.

File: system/README.md (hunk 1 above)
Comment (Mesbah-Alam) [reply system/README.md]: Fixed. I had missed a few more actually! Updated the PR with them too.

**Comment by Mesbah-Alam** [comment]
Note, the [Directories/Files change Test automated test ](https://github.com/adoptium/aqa-tests/pull/2542/checks?check_run_id=2547738169) is failing with : 

```
/home/runner/work/aqa-tests/aqa-tests/openjdk-tests/system/aqa-systemtest/openjdk.build/build.xml:48: Basedir /home/runner/work/aqa-tests/aqa-tests/openjdk-tests/system/openjdk-systemtest/openjdk.test.concurrent does not exist
```

This is due to the fact that this build is checking out aqa-systemtest from master: 

```
clone_systemtest:
     [echo] Cloning from: https://github.com/adoptium/aqa-systemtest.git -b master
     [exec] Cloning into 'aqa-systemtest'...
```

So, the [change](https://github.com/adoptium/aqa-systemtest/pull/425/files) associated with renaming path of systemtest projects is not available in the `openjdk.build/build.xml` that this build is executing, reaulting in the failure. 

@pshipton - can we ignore this check for this PR? Or is there a way we can make these automated builds point to the aqa-systemtest [developer branch](https://github.com/adoptium/aqa-systemtest/pull/425/files)?

**Comment by llxia** [comment]
Please note all 4 PRs have to merged together:
adoptium/STF#115
adoptium/aqa-systemtest#425
eclipse-openj9/openj9-systemtest#129
adoptium/aqa-tests#2542

**Comment by llxia** [comment]
Please update https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/testJobTemplate#L228

**Review by llxia — APPROVED** [review]
LGTM

File: doc/Triage.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ There are many different test jobs running at the AdoptOpenJDK project.  No matt
...
+- test issue - ideally, there are enough details to determine which test repo to raise an issue in ond of the test repositories from which test material is pulled (OpenJDK, [aqa-systemtest](https://github.com/adoptium/aqa-systemtest/issues), [openjdk-tests](https://github.com/AdoptOpenJDK/openjdk-tests/issues) or any of the various 3rd party application suites).  If in doubt, ask some questions in the [#testing channel](https://adoptopenjdk.slack.com/messages/C5219G28G) and/or raise in [openjdk-tests](https://github.com/AdoptOpenJDK/openjdk-tests/issues) where it will get routed to proper repo
```
Comment (smlambert) [inline doc/Triage.md:29]: testing channel link -> https://adoptium.slack.com/archives/C5219G28G 

[openjdk-tests](https://github.com/AdoptOpenJDK/openjdk-tests/issues) -> [aqa-tests](https://github.com/adoptium/aqa-tests/issues)

**Review by smlambert — COMMENTED** [review]
Couple of minor updates for slack/github links, otherwise LGTM.

File: doc/Triage.md (hunk 4 above)
Comment (Mesbah-Alam) [reply doc/Triage.md]: Thanks for spotting those! Updated!

**Review by smlambert — APPROVED** [review]
thanks @Mesbah-Alam !

**Pull request merged**